classifier/__init__.py

```diff
@@ -1281,6 +1323,12 @@ def audience(self, genres=[], default_audience=None):
             # combined they do pass the threshold. Go with
             # 'Young Adult' to be safe.
             audience = Classifier.AUDIENCE_YOUNG_ADULT
+        elif (research_weight > (total_adult_weight + all_ages_weight) and
+            research_weight > (total_juvenile_weight + all_ages_weight)):
```

Comment:
I'm still not sure if this is quite what we want - if juvenile weight passes the threshold, we wouldn't hit this elif even if the research weight is much greater.

Reply:
I added an example in the tests where this occurs and the audience is research. Maybe it's not as accurate of an example?

Reply:
I think this setup is what Amy has in mind:

adult=100
juvenile=150
research=1000

Reply:
I see. This can work if the research clause is checked before any of the other clauses, but should research have more of a priority than the other weights?

Reply:
Yeah, check the research hypothesis first, and it has to outweigh every other hypothesis. (be higher than the children threshold, and the ya threshold, and the adult threshold, and the all ages threshold)